umbra-js/.nycrc

```diff
@@ -0,0 +1,25 @@
+{
+  "cache": false,
+  "check-coverage": false,
+  "extension": [
+    ".ts"
+  ],
+  "include": [
+    "**/*.js",
+    "**/*.ts"
+  ],
+  "exclude": [
+    "coverage/**",
+    "node_modules/**",
+    "**/*.d.ts",
+    "**/*.test.ts"
+  ],
```

Comment:
Looking at the results of coverage, it says the `umbra-js/types/contracts` folder has a lot of missing coverage. Should this folder be excluded since it's just typechain type files?